src/macroexpand.scm

```diff
@@ -407,10 +407,30 @@
                      (and (eq? (car e) '=) (length= e 3)
                           (eventually-call? (cadr e))))))
 
+;; count hygienic / escape pairs
+;; and fold together a list resulting from applying the function to
+;; and block at the same hygienic scope
```

Comment:
is this supposed to say "applying the function to a block at the same hygienic scope" ?